frontend/lib/src/components/widgets/ColorPicker/ColorPicker.tsx

```diff
@@ -72,13 +72,13 @@ const updateWidgetMgrState = (
   )
 }
 
-const ColorPicker: FC<Props> = ({
+const ColorPicker: FC<Props> = memo(function ColorPicker({
```

Comment:
nit: I think most of our elements that already use memo have it wrap around on the bottom, e.g.:
`export default memo(ColorPicker)` in this case. Not sure if there is any benefits or downside to any of the approaches, but it might be good to keep it one way or another for consistency.